tests/testthat/test-rd-family.R

```diff
@@ -95,3 +95,31 @@ test_that("family also included in concepts", {
 
   expect_equal(out$get_field("concept")$values, "a")
 })
+
+test_that("custom family prefixes can be set", {
+
+  owd <- setwd(tempdir())
+  on.exit(setwd(owd), add = TRUE)
+
+  dir.create("man/roxygen", recursive = TRUE)
+  write_lines(
+    "list(rd_family_title = list(a = 'Custom prefix: '))",
+    "man/roxygen/meta.R"
+  )
+
+  roxy_meta_load(getwd())
```

Comment:
Maybe it would better to have `roxy_meta_set()` that you could pass a list? Then have a separate set of tests for `roxy_meta_load()`?